### ISSUE TITLE
Fine-tune with my own dataset, wer is 1

### ISSUE THREAD

**Issue by abcdbosh** [TencentGameMate/chinese_speech_pretrain#22]
finetune according to this blog： https://huggingface.co/blog/fine-tune-wav2vec2-english，
But this problem arises，Here it is train_state.json:
 "best_metric": null,
  "best_model_checkpoint": null,
  "epoch": 18.9873417721519,
  "global_step": 1500,
  "is_hyper_param_search": false,
  "is_local_process_zero": true,
  "is_world_process_zero": true,
  "log_history": [
    {
      "epoch": 6.33,
      "learning_rate": 3.49e-05,
      "loss": 14.8414,
      "step": 500
    },
    {
      "epoch": 6.33,
      "eval_loss": NaN,
      "eval_runtime": 36.5218,
      "eval_samples_per_second": 49.286,
      "eval_steps_per_second": 2.054,
      "eval_wer": 0.9999660072064722,
      "step": 500
    },
    {
      "epoch": 12.66,
      "learning_rate": 8.34e-05,
      "loss": 0.0,
      "step": 1000
    },
    {
      "epoch": 12.66,
      "eval_loss": NaN,
      "eval_runtime": 36.5778,
      "eval_samples_per_second": 49.21,
      "eval_steps_per_second": 2.05,
      "eval_wer": 0.9999660072064722,
      "step": 1000
    },
    {
      "epoch": 18.99,
      "learning_rate": 7.562043795620438e-05,
      "loss": 0.0,
      "step": 1500
    },
    {
      "epoch": 18.99,
      "eval_loss": NaN,
      "eval_runtime": 36.5273,
      "eval_samples_per_second": 49.278,
      "eval_steps_per_second": 2.053,
      "eval_wer": 0.9999660072064722,
      "step": 1500
    }
  ],
  "max_steps": 2370,
  "num_train_epochs": 30,
  "total_flos": 4.074416087326559e+18,
  "trial_name": null,
  "trial_params": null
}

What could be the problem?
